app/views/users/show.html.erb

```diff
@@ -1,12 +1,36 @@
-<h1>Users#show</h1>
-<p>Find me in app/views/users/show.html.erb</p>
-<!-- app/views/users/index.html.erb -->
-<h1>All Users</h1>
-<p>Here is a list of all users:</p>
-<!-- Implement code to display the list of users -->
-
-<!-- app/views/users/show.html.erb -->
-<h1>User Details</h1>
-<p>Here are the details for the user:</p>
-<!-- Implement code to display the details of a single user -->
-
+<h1 class='title'>Welcome to Blog-Haven</h1>
+<div class= "container">
+<% if @user %>
+    <div class='user'>
+      <div class='avatar'>
+        <%= image_tag(@user.photo) %>
+      </div>
+    <div><h2><%= @user.name %></h2></div>
+    <div><p>Number of Posts: x</p></div>
+    </div>
+    <div class='user-info'>
+      <div><h2>Bio</h2></div>
+      <div><p><%= @user.bio %><br/>Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat aliquid eius fuga. Nulla reprehenderit, ipsa quam, asperiores itaque totam vel amet iste ducimus pariatur distinctio reiciendis quasi hic, illum sit.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Assumenda, quas magni. Iure deserunt voluptates accusantium quam maxime tempore sint veniam laudantium? Aperiam placeat dolor quasi facilis! Dicta quasi soluta fugiat cupiditate facere consequuntur ipsum! Ea beatae, adipisci delectus sunt rerum veniam velit odio sit deleniti deserunt nobis facere nostrum ex!</p>
+      </div>
+      </div>
```

Comment:
- [ ] As the previous reviewer mentioned kindly fix the post count and also match the given wireframe 
in your design :point_down: 
![Screenshot from 2023-07-21 23-39-39](https://github.com/eudondian/Blog-Haven/assets/66097371/832928ad-c008-47ec-8e31-1e67837089f6)
given wireframe :point_down: 
![Screenshot from 2023-06-14 23-02-12](https://github.com/eudondian/Blog-Haven/assets/66097371/cfb86dbd-c74d-493e-ace3-e6743c7df9ba)

Reply:
well resolved! ✔️
Design follows the wireframe though there's not bother applied to design.